README.md

```diff
@@ -170,6 +176,101 @@ That's it!
 
 - [UniRx](https://github.com/neuecc/UniRx) is refered on Example
 
+# API
+
+## `StateBase`
+
+```csharp
+public class State : StateBase<State>
+{
+    public int Count { get; set; }
+}
+```
+
+### `<StateBase>.Clone()`
+
+```csharp
+State _state = new State();
+State _clonedState = _state.Clone();
+```
+
+Create a deep clone of the current state. Useful for Immutability.
+
+## `Store`
+
+```csharp
+Store _store = new Store<State>(State);
+// State must extend StateBase
+```
+
+### `<Store>.State`
+
+Get the state as passed to the constructor.
+
+### `<Store>.AddReducer<A>(<R>)`
+
+Add a Reducer which handles events of type `A`.
+Only one reducer per type is allowed.
```

Comment:
I thought `A` is not easy to understand.  I'll fix it to replacing from `A` to `TAction` (including original source). https://github.com/mattak/Unidux/issues/67